policyengine_us/parameters/gov/states/vt/tax/income/main/head_of_household.yaml

```diff
@@ -0,0 +1,35 @@
+description: Vermont taxes income of head of household filers at this rate. 
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Vermont income tax rates head of household filers
+  reference:
+    - title: Vermont §5822. Tax on income of individuals, estates, and trusts
+      href: https://legislature.vermont.gov/statutes/section/32/151/05822
+    - title: Vermont 2022 Income Tax Return Form N-111
```

Comment:
this form does not mention the parameters below - please find a representative tax form that shows the tax rate

Reply:
[Line 12-13](https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf) (Page 45) is for 2022 and [Line 14-15](https://tax.vermont.gov/sites/tax/files/documents/RateSched-2021.pdf) is for 2021. I included lines 8-11 because Nathan found those when he created [issue #2439](https://github.com/PolicyEngine/policyengine-us/issues/2439). We can consider deleting line 8-11 if line 12-15 are enough.

Reply:
the lines work if they point to the parameter, just need to add it to the 2021 reference pdf